_run/lite/Makefile

```diff
@@ -3,24 +3,30 @@ include ../common.mk
 akashctl       = $(AKASHCTL) --home $(CLIENT_HOME) 
 akashd         = $(AKASHD) --home $(NODE_HOME)
 KEY_OPTS       = --keyring-backend=test
-CHAIN_NAME     = local
-CHAIN_OPTS     = --chain-id $(CHAIN_NAME)
 
 init: bins init-keys init-genesis init-gentx init-collect-gentx init-config
```

Comment:
add `init-node`?

Reply:
probably but I'm not sure.  have to reconcile w/ other `_run/` dirs.